### DISCUSSION TITLE
Framework API Response not visible

### DISCUSSION THREAD

**Discussion by hpDeviOS** [Flipboard/FLEX#612] [Q&A]
I am working on the iOS Framework and App, both have API calls.
The framework used in App.

So, whenever I use FLEX in App, whatever API calls under the framework its response receive successfully but is **NOT VISIBLE in flex**. (showing awaiting for response).

This issue only when API calls from Framework, no issue for API calls from App.

![IMG_9EA726D5BC05-1](https://user-images.githubusercontent.com/22273964/179837803-4c854661-e4fc-4acd-bbc4-d6883b04d0aa.jpeg)

Is there any configuration that needs to make in App or Framework?
or
anything else.

**Comment by NSExceptional** [comment]
This is a known issue I've struggled to debug. Which framework is it? It would be helpful if I could test it myself

**Reply by hpDeviOS** [reply]
It's custom framework, own developed.

**Reply by ArthurAvagyan** [reply]
Hi, I have installed FLEX today and I'm having this kind of an issue, but only I'm not able to see my app's requests. I'm not using any kind if 3rd party libs for requests, I use simple URLSession. My guess is that the URL I am requesting to is http and not https, am I right? And where can I find more documentation on FLEX rather then the README file?:)

**Reply by NSExceptional** [reply]
Are you using async/await? Can you post any code?
